docs/developer-guide/standardize-python-project-configuration.md

```diff
@@ -0,0 +1,99 @@
+---
+myst:
+  html_meta:
+    "description": "Standardize project configuration in Plone with meta"
+    "property=og:description": "Standardize project configuration in Plone with meta"
+    "property=og:title": "Standardize project configuration in Plone with meta"
+    "keywords": "Plone 6, standardize, project, configuration, development, meta"
+---
+
+# Standardize Python project configuration
+
+This part of the documentation describes how to standardize Python project configuration in Plone.
+It does not cover Volto or any other JavaScript-based project, which has its own ecosystem.
+
+Plone consists of hundreds of projects.
+To lessen the effort of configuring a new project in the Plone GitHub organization, and to keep these projects current with latest configuration practices, the Plone community agreed upon a trusted set of configuration items.
+The Plone community manages these configuration items using the [`meta`](https://github.com/plone/meta) project.
+
+You can follow these practices in your own projects, or suggest new or alternative configuration items through the `meta` repository, sharing them with the rest of the Plone community.
+
+
+## `meta` basic usage
+
+`meta` has a rich set of features.
+This section describes the most common use cases.
+
+```{seealso}
+See a description of all of [`meta`'s features](https://github.com/plone/meta/blob/main/config/README.md#quick-start).
+```
+
+
+### Setup
+
+Clone `meta` to any machine, then change your current working directory into `meta/config`, create a Python virtual environment, activate it, and install `meta`'s requirements.
+
+```shell
+git clone https://github.com/plone/meta.git
+cd meta/config
+python3 -m venv venv
+. venv/bin/activate
```

Comment:
We generally avoid telling people to activate a virtualenv in the docs, because if they don't understand what they're doing it can lead to tricky situations to troubleshoot (i.e. they try to run commands in a new terminal where it isn't activated). So better to leave it inactive and give full paths to the commands in the virtualenv.

Reply:
I agree. This was a copy-paste from meta itself. I'll update it here, but I'd like @gforcada to bless this change and consider reusing it upstream.

Done in 302d6e25d4b015a97dde403c88fe5fcd0b112b8d.